http/headers/keep-alive.json

```diff
@@ -6,7 +6,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Headers/Keep-Alive",
           "spec_url": [
             "https://datatracker.ietf.org/doc/html/draft-thomson-hybi-http-timeout-03#section-2",
-            "https://httpwg.org/specs/rfc7230.html#appendix-A.1.2"
+            "https://httpwg.org/specs/rfc7230.html#compatibility.with.http.1.0.persistent.connections"
```

Comment:
Sorry about the merge conflict here. I had removed the first datatracker url

Reply:
OK